docs/sources/send-data/otel/_index.md

```diff
@@ -16,14 +16,21 @@ For ingesting logs to Loki using the OpenTelemetry Collector, you must use the [
 
 When logs are ingested by Loki using an OpenTelemetry protocol (OTLP) ingestion endpoint, some of the data is stored as [Structured Metadata]({{< relref "../../get-started/labels/structured-metadata" >}}).
 
+*Make sure to set the `allow_structured_metadata` to `true` within your Loki config file. Otherwise, Loki will reject the log payload as malformed.*
```

Comment:
Our style guide says not to use single asterisks for emphasis.
```suggestion
You must set `allow_structured_metadata` to `true` within your Loki config file. Otherwise, Loki will reject the log payload as malformed.
```